packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3482,15 +3491,22 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     return true;
   }
 
+  @deprecated
```

Comment:
Do you have to repeat the deprecation notice in the override as well? That seems odd...

If we do, this should be the full notice that tells you what to use instead.

Reply:
Without you get a warning when you call an other deprecated method in it's body.

Reply:
I was just playing around with this: It looks like we need to repeat the deprecation notice on all overridden methods as well, otherwise users calling the overridden method directly will not see the deprecation. And the repeated deprecation should also contain what you should call instead.